### ISSUE TITLE
Required Decorators Analysis

### ISSUE THREAD

**Issue by jonathan-casarrubias** [strongloop/loopback4-extension-grpc#6]
> API-first (top-down) vs. code-first (bottom-up) approach, how to provide protobuf metadata - think of @api and @get decorators for REST

https://github.com/strongloop/loopback-next/issues/582

As developer, I want be able to use TypeScript decorators as an option to add proto services.

In order to follow a naming convention from the official gRPC module, I propose to use addProtoService name for this decorator, so it is easier for people coming to LB4 and that already knows gRPC to ramp up

**Comment by jonathan-casarrubias** [comment]
If we want to support microservices oriented architecture... This might be a good implementation

````js
@gRPCService({ host: ‘’, port: 9000, proto: '' })
class MyController {
    @rpc()
    sayHello(HelloRequest): HelloResponse {}
}
````

**Comment by jonathan-casarrubias** [comment]
Landed on PR #8
